src/test/e2e/docker-compose.yml

```
@@ -0,0 +1,200 @@
+    networks:
+      - cloud-update-test
+    environment:
+      - E2E_SECRET=test-secret-key-for-e2e
+    volumes:
+      - test-results:/results
+    command: |
+      sh -c "
+        echo 'Waiting for services to be ready...'
+        sleep 5
+        echo 'Running E2E tests for all distributions...'
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)